fuel-core/src/args.rs

```diff
@@ -137,3 +178,34 @@ impl Opt {
         })
     }
 }
+
+pub fn dump_snapshot(path: PathBuf, _config: ChainConfig) -> anyhow::Result<()> {
```

Comment:
config is unused if rocksdb isn't enabled because a snapshot couldn't be taken, and so it's never used. I got around this with path by printing it in the error, would you like me to do the same with config?

Reply:
Ah makes sense, I think it's fine as is then.

Reply:
Usually, you specify two functions one with an error and one without it and use cfg to enable/disable them.